paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -108,7 +108,13 @@ class PaymentSheet internal constructor(
          *
          * If set, PaymentSheet will pre-populate the form fields with the values provided.
          */
-        var defaultBillingDetails: BillingDetails? = null
+        var defaultBillingDetails: BillingDetails? = null,
+
+        /**
+         * Whether or not to allow payment methods that have a delay before payment or setup
+         * completes, like most bank debits (e.g SEPA) and vouchers (e.g. OXXO, Konbini, Boleto).
```

Comment:
Please use this comment 

> If `true`, allows payment methods that do not move money at the end of the checkout. Defaults to false.
> 
> Description: Some payment methods can't guarantee you will receive funds from your customer at the end of the checkout because they take time to settle (eg. most bank debits, like SEPA or ACH) or require customer action to complete (e.g. OXXO, Konbini, Boleto). If this is set to `true`, make sure your integration listens to webhooks for notifications on whether a payment has succeeded or not.
> Seealso: https://stripe.com/docs/payments/payment-methods#payment-notification